### PR TITLE
swig has been updated to 3.0.6

### PR DESCRIPTION
**Pull request by hans2103**
The SWIG project has released a new version 3.0.6
https://github.com/swig/swig/releases

### CODE REVIEW

File: Library/Formula/swig.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Swig < Formula
   desc "Generate scripting interfaces to C/C++ code"
   homepage "http://www.swig.org/"
-  url "https://downloads.sourceforge.net/project/swig/swig/swig-3.0.5/swig-3.0.5.tar.gz"
+  url "https://downloads.sourceforge.net/project/swig/swig/swig-3.0.6/swig-3.0.6.tar.gz"
   sha1 "271813b317e4836853d2249fc8ce2df34c2a062a"
```
Comment (mikemcquaid) [inline Library/Formula/swig.rb]: This will need updated (ideally to a `sha256`)

Comment (hans2103) [inline Library/Formula/swig.rb]: I've never done that. Can you tell me how or create a pull request on my pull request?

Comment (mikemcquaid) [inline Library/Formula/swig.rb]: `brew fetch swig` locally and edit the `sha1` field to be `sha256` and whatever output it showed.

**Comment by hans2103** [comment]
thx

**Pull request closed without merging**